receiver/aerospikereceiver/mocks/Aerospike.go

```diff
@@ -0,0 +1,77 @@
+// Code generated by mockery v2.12.1. DO NOT EDIT.
```

Comment:
This does use `github.com/stretchr/testify/mock`, but mockery is just the tool that generates the boilerplate from a given interface. I can update the Makefile so that it gets installed

Reply:
Is it necessary to use mockery? I believe it's been avoided in the past.

Reply:
It's not necessary, the benefit I see from it is just that the mock functions really are boilerplate with only types changing. But I can rewrite these myself if we don't want to depend on Mockery